content/en/docs/reference/glossary/cluster.md

```diff
@@ -11,10 +11,12 @@ tags:
 - fundamental
 - operation
 ---
-A set of worker machines, called {{< glossary_tooltip text="nodes" term_id="node" >}},
-that run containerized applications. Every cluster has at least one worker node.
+A set of machines, called {{< glossary_tooltip text="nodes" term_id="node" >}}, that run
+containerized applications managed by Kubernetes. To have a working cluster you need a
```

Comment:
nit: There is a difference (discrepancy?) between the short and long descriptions. For example:
`A set of machines called nodes` vs `A set of worker machines called nodes`
What about the control plane (does every cluster need a control plane)?

Reply:
The difference between long and short description is by intention. I'll look at the other details too.

Reply:
👍